src/schema/SearchableItem/SearchableItemPresenter.ts

```diff
@@ -102,11 +102,22 @@ export class SearchableItemPresenter {
     return this.item.image_url
   }
 
-  private formattedEventDescription(title: string): string {
+  private formattedEventDescription(title: string, timezone?: string): string {
     const { description, location, start_at, end_at } = this.item
 
-    const formattedStartAt = moment.utc(start_at).format(DATE_FORMAT)
-    const formattedEndAt = moment.utc(end_at).format(DATE_FORMAT)
+    const startAt = moment.utc(start_at)
+    const endAt = moment.utc(end_at)
+    let formattedStartAt = startAt.format(DATE_FORMAT)
+    let formattedEndAt = endAt.format(DATE_FORMAT)
+
+    if (timezone) {
+      formattedStartAt = `${formattedStartAt} (at ${startAt
+        .tz(timezone)
+        .format("h:mma z")})`
```

Comment:
We've had bugs where `moment(null).tz(...).format(...)` results in `Invalid Date` (recently an example was fixed in https://github.com/artsy/metaphysics/pull/1656).

Because down below, we're guarding against `start_at` or `end_at` being null, that means that these strings will become `Invalid Date`. Now, most likely these aren't being referenced or returned after this, but it still feels a little weird.

Perhaps add a spec that has one/both of `start/end_at` as `null`, just to make sure everything is all good?

Reply:
Thanks for context! Great suggestion. I added additional specs around only having `start_at`. We already had one exercising both with `null` values. 

Also refactored some of this timestamp handling to more explicitly handle the Invalid Date handling from moment.js.